chef/templates/default/manifest.xml.erb

```diff
@@ -15,7 +15,7 @@
             <service_fmri value="svc:/milestone/multi-user"/>
         </dependency>
         <exec_method timeout_seconds="60" type="method" name="start"
-            exec="<%= @command.encode(:xml => :text) %>">
+            exec="<%= "nohup #{@command} &".encode(:xml => :text) %>">
```

Comment:
(this is bad I just don't have the cycles to fix it this second, this should work more like the monit plugin where it uses the init.d script generation to get more stable process backgrounding (probably).